lib/async_hooks.js

```diff
@@ -516,3 +517,10 @@ Object.defineProperty(module.exports, 'triggerId', {
   }, 'async_hooks.triggerId is deprecated. ' +
      'Use async_hooks.triggerAsyncId instead.', 'DEP0071')
 });
+
+// Integration with V8 Inspector AsyncTask* API
+// FIXME(bajtos) move this code to src/inspector_agent.cc
+// Agent::Start() should enable the hook, Agent::Stop() should disable it
```

Comment:
I would consider this FIXME a blocker for this PR, let me know if you need any help with it.